### PR TITLE
enable travis CI

### PR DESCRIPTION
**Pull request by heartlock**
Signed-off-by: heartlock <21521209@zju.edu.cn>
ref: #4 
@Random-Liu  Please enable travis hook and test.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - 1.8.x
+    - tip
+
+sudo: required
+
+install:
+    - make install.tools
+
+before_script:
+    - export PATH=$HOME/gopath/bin:$PATH
```
Comment (mikebrow) [inline .travis.yml]: $HOME/gopath/bin:$PATH

is that correct?

Comment (mikebrow) [inline .travis.yml]: check if this is needed after @Random-Liu enables travis

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+language: go
+go:
+    - 1.6.x
```
Comment (cmluciano) [inline .travis.yml]: Does Kube main still test on 1.6?

Comment (heartlock) [inline .travis.yml]: How to check whether kube main still test on golang 1.6 @cmluciano

File: .travis.yml (hunk 1 above)
Comment (Random-Liu) [inline .travis.yml]: I'm not familiar with travis. I should have enabled travis for this project, we could try.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+sudo: required
+
+install:
+    - make install.tools
+
+before_script:
+    - export PATH=$HOME/gopath/bin:$PATH
+
+script:
+    - make .gitvalidation
```
Comment (Random-Liu) [inline .travis.yml]: It seems that the commit 022b25259e31784d68192cdcd87a82d6c357a1bb doesn't have DCO, and will cause error if we run locally.

@mikebrow For this, should we update the previous commit or skip it? Maybe we should change https://github.com/kubernetes-incubator/cri-containerd/blob/master/Makefile#L2 to 022b25259e31784d68192cdcd87a82d6c357a1bb.

**Comment by Random-Liu** [comment]
Close and reopen to trigger the travis bulid.

File: .travis.yml (hunk 3 above)
Comment (mikebrow) [inline .travis.yml]: Correct!  I'd do it but on my phone ATM.

Comment (mikebrow) [inline .travis.yml]: By correct I mean change the base commit on line 2 to the first one that passes

Comment (Random-Liu) [inline .travis.yml]: @heartlock Could you make the change in your PR? Change the gitvalidation base commit to 022b25259e31784d68192cdcd87a82d6c357a1bb

Comment (heartlock) [inline .travis.yml]: sure, I will update

Comment (mikebrow) [inline .travis.yml]:
```
EPOCH_TEST_COMMIT ?= 022b25259e31784d68192cdcd87a82d6c357a1bb
PROJECT := github.com/kubernetes-incubator/cri-containerd
```

**Comment by heartlock** [comment]
@Random-Liu @mikebrow  updated

**Comment by Random-Liu** [comment]
LGTM

**Comment by Random-Liu** [comment]
@heartlock Thanks for your help!

**Pull request merged**